src/main/CommandLine.cpp

```diff
@@ -1041,30 +1041,109 @@ runReportLastHistoryCheckpoint(CommandLineArgs const& args)
         });
 }
 
+std::vector<std::pair<uint32_t, uint32_t>>
```

Comment:
we should be returning `std::chrono::microseconds`

Reply:
Updated it.